readme.md

```diff
@@ -77,6 +81,64 @@ modules: [
 ]
 ```
 
+and alternative approach, is to define a fixed MMM-Pages config
+
+```js
+modules: [
+    {
+        module: 'MMM-pages',
+        config: {
+                modules:
+                    [[ "page1" ],
+                     [ "page2" ],
+                     [ "page3" ],
+                    ],
```

Comment:
```suggestion
                     [ "page3" ]],
```

Reply:
Ps, rude. Could have said typo

Reply:
Hey, I was just showing an example of where I meant by "matching the formatting of the previous example".

It's not a typo, it's just a format request.

Reply:
The proper english, was rude

I changed the formatting in a different way
I edited the file

I like to expose the outside brackets as many users don't see them, and make mistakes
Tte trailing comma was irrelevant

Reply:
Sam,

I don't mind you communicating to me in whatever style you prefer, but documentation communicates to everyone that sees the module.

There are people that use this module that might not understand English as well or even at all. They might be learning English or be using a machine translator, and not using proper English makes it harder for them to use the module. I've even had to use my terrible proficiency of French to help out some users in the past.

I ask for proper English because the existing documentation uses proper English. I'm asking you to match the style of what already exists, and to make it easier for our international users to use the module.

I don't understand how that would be considered rude by any means.